packages/api-client/src/conversation/ConversationAPI/ConversationAPI.ts

```diff
@@ -66,11 +67,10 @@ import {
   ConversationReceiptModeUpdateData,
   ConversationTypingData,
 } from '../data';
-import {MlsEvent} from '../data/MlsEventData';
 import {Subconversation, SUBCONVERSATION_ID} from '../Subconversation';
 
 export type PostMlsMessageResponse = {
-  events: MlsEvent[];
+  events: BackendEvent[];
```

Comment:
When sending a commit bundle, the backend can reply with any kind of backend event. It's not directly linked to MLS